mappings/net/minecraft/inventory/Inventory.mapping

```diff
@@ -1,4 +1,12 @@
-CLASS none/ri
+CLASS none/ri net/minecraft/inventory/Inventory
+	FIELD a name Ljava/lang/String;
+	FIELD b size I
+	FIELD c stackList Lnone/dc;
+	FIELD e hasCustomName Z
```

Comment:
`has` is supposed to be reserved for methods. maybe `customNamed`, or just `customName`, since it is a bool? Though I guess an exception makes the most sense here.